telegram/_message.py

```diff
@@ -1556,6 +1556,23 @@ def _quote(
     def compute_quote_position_and_entities(
         self, quote: str, index: Optional[int] = None
     ) -> Tuple[int, Tuple[MessageEntity, ...]]:
+        """
+        Use this function to compute position and entities of quote.
+
+        Args:
+            quote (:obj:`str`): Part of the message which is to be quoted. This is
+                expected to have plain text without formatting entities.
+            index (:obj:`int`, optional): Position of the quote in the original message in
+                UTF-16 code units.
```

Comment:
This one is still not right. What we want here is the number of the appearance of the text. E.g. if the message has the text

> ABC DEF ABC DEF

calling `compute_quote_position("ABC")` or `compute_quote_position("ABC", 0)` should quote the first "ABC", but calling `compute_quote_position("ABC", 1)` should quote the second "ABC" (standard zero-based indexing)

the UTF-16 offset will have to be computed and is part of the output. Computing this for the user is an important part of this convenience functionality b/c emojis & stuff are not quite as easy to handle in utf-16. that's also why the "apply utf-16 conversion" is still open below ;) have a look at `_parse_html/md_entities`, where utf-16 handling is used as well

Reply:
Hmm should we also add examples regarding the that to docstrings?